pandas/_libs/src/parser/tokenizer.c

```diff
@@ -248,7 +248,8 @@ void parser_del(parser_t *self) {
 }
 
 static int make_stream_space(parser_t *self, size_t nbytes) {
-    int64_t i, cap, length;
+    uint64_t i, cap;
+    int64_t length;
```

Comment:
why is length signed?

Reply:
Because there weren't any warnings telling us not to.  Probably makes more sense unsigned liked the others.

Reply:
can you change this, otherwise lgtm.

Reply:
Changing this requires changing a few other places to match.  I think its benign, but not obvious.  Go for it?

Reply:
yep prob should